src/bin/ion/commands/inspect.rs

```diff
@@ -243,28 +275,36 @@ impl<'a, 'b> IonInspector<'a, 'b> {
                 InspectorAction::LimitReached => break,
             }
 
-            if !is_first_item && !is_last_item {
+            if !is_first_item && !is_last_item && !expr.is_ephemeral() {
                 // If this item is neither the first nor last in the stream, print a row separator.
                 write!(self.output, "{ROW_SEPARATOR}")?;
             }
 
-            match item {
-                SystemStreamItem::SymbolTable(lazy_struct) => {
+            match expr {
+                EExp(eexp) => {
+                    self.inspect_eexp(0, eexp)?;
```

Comment:
🗺️ We now display e-expressions found at the top level. Some more work is needed in `ion-rust` to make it possible to get an 'expanded' view of lists and structs, allowing us to inspect both the encoded expressions and the values/fields they produce.